package.json

```diff
@@ -202,6 +203,7 @@
     "command-exists": "1.2.9",
     "conf": "^7.0.1",
     "crypto-js": "^4.0.0",
+    "deep-equal": "^2.0.5",
```

Comment:
How it differs from `lodash/isEqual`?

Reply:
This package uses node's deep equal algorithm, but I'll switch it out for the lodash one.